src/main/surfrdMod.F90

```diff
@@ -475,9 +481,9 @@ subroutine surfrd_special(begg, endg, ncid, ns)
 
     topo_glc_mec(:,:) = max(topo_glc_mec(:,:), 0._r8)
 
-    pctspec = pctwet + pctlak + pcturb_tot + pctgla
+    pctspec = pctwet + pctlak + pcturb_tot + pctgla + pctocn
```

Comment:
In reviewing this with @swensosc:

Sean asked whether this code should be backwards compatible with previous fsurdat files that do not include pct_ocean.

As written, the code is not backwards compatible, and maybe we need to consider this.

Reply:
> In reviewing this with @swensosc:
> 
> Sean asked whether this code should be backwards compatible with previous fsurdat files that do not include pct_ocean.
> 
> As written, the code is not backwards compatible, and maybe we need to consider this.

@ekluzek @billsacks 
Sean and I didn't have strong opinions on this. I, in particular, was not worried about backwards compatibility between ctsm5.2 fsurdat files and ctsm5.1 code, because we already need a short list of hacks for such backwards compatibility, so this could be added to the list if necessary.

I am interested in your feedback, and I can raise the question on Thursday morning if you prefer.

Reply:
I agree -- we really can't use CTSM5.2 fsurdat files with CTSM5.1 or vice versa. So I don't want to worry about backwards compatibility. It always adds complexity. and it's complexity that becomes less useful over time.

@swensosc is right to bring it up though, because we should think about it. And in terms of CTSM5.1 datasets we did try to keep them backwards compatible. And I think we'll try to do the same in the future, until we have another major change in datasets.

I also think that having PCT_OCEAN on the datasets is a real improvement in making the datasets understandable to users. PCT_WETLAND made users assume it really meant inland wetlands, and certainly not actually ocean. So these datasets with PCT_OCEAN is much easier for users to understand.

Reply:
I agree with Erik regarding backwards compatibility - specifically, skipping it in this instance.